web/pgadmin/about/__init__.py

```diff
@@ -123,6 +123,13 @@ def detect_browser(request):
     agent = request.environ.get('HTTP_USER_AGENT')
     os_details = parse(platform.platform()).ua_string
 
+    try:
```

Comment:
How is this related to AppArmor?

Reply:
It is not, I found it helpful to add in the about dialog.

Reply:
It should be in a separate PR then.

Reply:
OK. I'll remove the change.